web/source/kmwkeyboards.ts

```diff
@@ -1212,7 +1212,7 @@ namespace com.keyman {
         Lscript = this.keymanweb.util._CreateElement('script');
       
       URL = URL + ((arguments.length > 1) && byLanguage ? 'languages' : 'keyboards')
-        +'?jsonp=keyman.register&languageidtype=bcp47&version='+this.keymanweb['version']+'.'+KeymanBase['__BUILD__'];
+        +'?jsonp=keyman.register&languageidtype=bcp47&version='+this.keymanweb['version']+'.'+this.keymanweb['build'];
```

Comment:
Several of our platforms have had recent issues passing invalid versions to api.keyman.com

Can you add a regex to `+this.keymanweb['version']+'.'+this.keymanweb['build']`:
1. to ensure it's valid
2. generate a warning/error for troubleshooting when invalid?

Reply:
Re-reading https://help.keyman.com/developer/cloud/4.0/ on `version`
> The version of Keyman Engine, e.g. "10.0". If absent, uses "2.0". The server will filter keyboards that are available for that version of Keyman Engine. (See KeymanWeb Versioning for details on valid version numbers.)

Maybe we only need `#.#` in the query?
e.g.
https://api.keyman.com/cloud/4.0/keyboards?jsonp=keyman.register&languageidtype=bcp47&version=14.0&keyboardid=sil_euro_latin@no&timerid=7

Reply:
@mcdurdin Any preference?

Reply:
I think we only need `#.#` in the query.